static/js/place/chart_embed.tsx

```diff
@@ -191,6 +192,7 @@ class ChartEmbed extends React.Component<unknown, ChartEmbedStateType> {
         sourcesHeight +
         SOURCES_MARGIN
     );
+    svg.attr("class", CLASS_DC_CHART_EXIST);
```

Comment:
probably want to add this attribute after actually appending the svg to the dom in the onOpened function? https://github.com/datacommonsorg/website/blob/master/static/js/place/chart_embed.tsx#L301

and same for the svgXml case